### ISSUE TITLE
[12.0] Quotation viewed by customer logged and received multiple times

### ISSUE THREAD

**Issue by sa3m** [odoo/odoo#55843]
Impacted versions: 12.0

Steps to reproduce: Send a quotation with the link to the online quotation (standard email)

Current behavior: I can't tell what the customer is doing exactly but the notification "Quotation viewed by customer" is logged 5 times in the chatter and the subscribers receive them 5 times in french and 5 times in english.

Expected behavior: The notification should be logged only once and subscribers should received only once in their language.

Video/Screenshot link (optional):
<img width="457" alt="image" src="https://user-images.githubusercontent.com/301632/90110657-4b394280-dd99-11ea-9b36-19c57c6663b7.png">

**Comment by pedrobaeza** [comment]
We have a lot of customers in v12, and none of them has this case, so it seems something special, but not a code bug. Unless you give steps to reproduce the problem, this can't be solved and will be closed.

**Comment by sa3m** [comment]
I understand since I wasn't able to reproduce this case myself.
It's the second time it happens with 2 differents customers.
How could I provide more info ? What would be needed ?

**Comment by pedrobaeza** [comment]
Some reproducible steps to get to the problem: doing this and that, it leads to this problem.

**Comment by sa3m** [comment]
There is nothing specific in the user's process. Our issue seems to come from the customer's side. 
The logs are very close (from 12 seconds to 2 minutes).

**Comment by pedrobaeza** [comment]
Then there's the possibility of the customer refreshing the page several times, or navigating back and forward, provoking the controller to trigger. Let's close this issue until further evidence of a bug.

**Comment by sa3m** [comment]
If the reason is the customer refreshing the page or navigating, this is a bug since there is a check done to log it only once and it doesn't work. I doubt that because I tested it extensively.

**Comment by Yenthe666** [comment]
@pedrobaeza I can assure you that @sa3m is right but I haven't found reproducible steps either. One of my V13 customers has it too:
![image](https://user-images.githubusercontent.com/6352350/90121403-3c806a80-dd5c-11ea-9e40-5cf9938fbd28.png)

**Comment by pedrobaeza** [comment]
He has v12. I haven't experienced this in a broad base of v12 customers. We have less v13, but none of them has it also. Yenthe, you know better than anyone that not having steps to reproduce is the same effective as nothing, and keeping this issue opened without these steps is no adding value. Anyone experiencing the same problem coming to this issue is not going to have any answer. If they have the steps, then they can open a new issue with them. Do you agree?

**Comment by sa3m** [comment]
Having steps for customer behavior is really hard to get, you never know what they are really doing and it could be dependant of their software as well.
This steps-to-reproduce thing shouldn't blind you from a potential issue.

> He has v12

Meaning it's reproduced on 2 versions ?

> Anyone experiencing the same problem coming to this issue is not going to have any answer.

 Sure, you directly closed the issue meaning this won't ever be treated if we don't find an answer ourselves.
Letting it opened might collect the same issue from different people.

**Comment by pedrobaeza** [comment]
Finding the answer is not the same as describing the steps to reproduce the problem. If you have an enteprise contract, just report the problem in odoo.com/help, and there, the OPW team can experiment even in your installation to find the steps, but this issue tracker is not for investigating that.

**Comment by Yenthe666** [comment]
I agree we need reproducable steps but there is a case where it happens though. I don't have the time to dig deeper now, if I do I'll come back.
